modules/project-docs/pages/sdk-full-installation.adoc

```diff
@@ -57,14 +57,14 @@ Configure tap for Couchbase:
 
 [source,console]
 ----
-$ brew tap couchbaselabs/homebrew-couchbase
+brew tap couchbaselabs/homebrew-couchbase
```

Comment:
$ and # prompts are put in our docs as visual aids,
but are filtered out by the copy-to-clipboard JS>